torch_geometric/loader/imbalanced_sampler.py

```diff
@@ -33,10 +33,22 @@ class distribution.
                                 batch_size=64, num_neighbors=[-1, -1],
                                 sampler=sampler, ...)
 
+    It is also supported to accept a :class:`torch.Tensor` object,
+    which is the node label that represents the class distribution:
```

Comment:
```suggestion
    which has the node label distribution that we wish to sample with:
```
Is this accurate? (No need to accept this change, but the current description was confusing for me)

I would have thought you'd directly want to provide weights in this case.